Tests/KernelTests/main.mm

```diff
@@ -57,6 +57,9 @@
       exit(1); \
     } while (0)
 
+const char *ldPath =
```

Comment:
This is brittle. Instead of hardcoding, use this:

```xcrun -f ld```, which prints the full path to the _active_ ld

Reply:
Done.